src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Http/HttpClientHandler/HttpClientHandlerCommon.cs

```diff
@@ -45,7 +45,10 @@ internal class HttpClientHandlerCommon
 
             try
             {
-                scope.Span.SetHttpStatusCode(responseMessage.StatusCode, isServer: false);
+                if (responseMessage.Instance != null)
+                {
+                    scope.Span.SetHttpStatusCode(responseMessage.StatusCode, isServer: false);
+                }
```

Comment:
Maybe we should revert this in favor of this PR that refactor and handles this scenario already: https://github.com/DataDog/dd-trace-dotnet/pull/1258

Reply:
Done